h2o-r/demos/rdemo.loo_encode.R

```diff
@@ -0,0 +1,60 @@
+library(h2o)
+h2o.init()
+
+airlinesPath <- h2o:::.h2o.locate("smalldata/airlines/AirlinesTrain.csv.zip")
+print("Importing airlines data into H2O")
+airlines.hex <- h2o.importFile(path = airlinesPath, destination_frame = "airlines.hex")
+
+print("Print out summary of airlines data")
+print(summary(airlines.hex))
+
+print("Split data into training and testing")
+splits.hex <- h2o.splitFrame(airlines.hex, seed = 1234, destination_frames = c("train.hex", "test.hex"))
+train.hex <- splits.hex[[1]]
+test.hex <- splits.hex[[2]]
+
+
+print("Run GBM without Leave One Out Encoding as Baseline")
+myX <- setdiff(colnames(train.hex), c("IsDepDelayed", "IsDepDelayed_REC", "fDayofMonth"))
+default.gbm <- h2o.gbm(x = myX, y = "IsDepDelayed", 
+                       training_frame = train.hex, validation_frame = test.hex, 
+                       ntrees = 100, score_tree_interval = 10, model_id = "default_gbm.hex",
+                       # Early Stopping
+                       stopping_rounds = 5, stopping_metric = "AUC", stopping_tolerance = 0.001)
+
+print(paste0("Default GBM AUC: ", round(h2o.auc(default.gbm, valid = TRUE), 3)))
+
+print("Peform LOO Encoding on Origin and Destination")
+
+for(i in c("Origin", "Dest")){
```

Comment:
Ok, maybe this comment doesn't make sense since I realized that you can create a single target encoding using either a single column or multiple columns.  So if you wanted to do a bulk creation of  target encodings (one for each column), there would be no way to distinguish between a single target encoding on all the columns vs a bunch of target encodings (one for each column).  Does that make sense?  So I think you can ignore this comment.